Tutorial.md

```diff
@@ -0,0 +1,66 @@
+# MSON Tutorial
+
+MSON is short for Markdown Syntax for Object Notation, and it's a way to represent data structures in a human readable plain text form. This tutorial will take you though the basics of MSON and how you can use MSON to describe a data structure.
+
+> **NOTE:** **Additional MSON Resources**
+>
+> + [MSON Specification](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md)
+> + [Examples](https://github.com/apiaryio/mson#notational-conventions)
+
+## Data structures in MSON
+
+To get started, we're going to create a data structure in MSON to represent a Person.
+
+A data structure to store a Person could be represented in MSON using the following:
+
+```apib
+# Person (object)
```

Comment:
In case of structures, object is the default type. In case of primitive
types, string is the default type. AFAIK

On Mon, Aug 10, 2015 at 12:11 PM, Kyle Fuller <notifications@github.com>
wrote:

> In Tutorial.md
> <https://github.com/apiaryio/mson/pull/34#discussion_r36617286>:
>
> > +
> > +MSON is short for Markdown Syntax for Object Notation, and it's a way to represent data structures in a human readable plain text form. This tutorial will take you though the basics of MSON and how you can use MSON to describe a data structure.
> > +
> > +> **NOTE:** **Additional MSON Resources**
> > +>
> > +> + [MSON Specification](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md)
> > +> + [Examples](https://github.com/apiaryio/mson#notational-conventions)
> > +
> > +## Data structures in MSON
> > +
> > +To get started, we're going to create a data structure in MSON to represent a Person.
> > +
> > +A data structure to store a Person could be represented in MSON using the following:
> > +
> > +```apib
> > +# Person (object)
>
> I though without it, the type would be a string?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apiaryio/mson/pull/34/files#r36617286>.
>